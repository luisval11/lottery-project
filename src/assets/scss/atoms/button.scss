@import "../abstracts/mixins";

button {
	background: linear-gradient(
		90deg,
		$color-button-01 0%,
		$color-button-02 100%
	);
	border-radius: 4px;
	box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.16);
	color: $color-ui-casino-01;
	font-size: 14px;
	line-height: 21px;
	text-align: center;
	@include CTA;
	min-height: 40px;
	padding: 7px 14px 7px 14px;
}
