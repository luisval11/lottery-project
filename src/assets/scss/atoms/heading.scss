@import "../abstracts/variables";

h4 {
	color: $color-ui-casino-02;
	font-feature-settings: "pnum" on, "lnum" on;
	font-size: 21px;
	@include CTA;
	line-height: 24px;
}

h5 {
	color: $color-ui-casino-02;
	font-feature-settings: "pnum" on, "lnum" on;		
	font-size: 17px;
	@include CTA;
	line-height: 21px;
	margin: 2px 0px;
}