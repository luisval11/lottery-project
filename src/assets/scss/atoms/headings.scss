@import "../abstracts/variables";

h5 {
    font-weight: 900;
    font-size: 17px;
    line-height: 21px;
    text-transform: uppercase;
    font-feature-settings: 'pnum' on, 'lnum' on;
    color: $color-ui-casino-02;
}

