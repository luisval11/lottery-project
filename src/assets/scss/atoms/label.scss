@import "../abstracts/variables";

label {
	color: $color-ui-casino-03;
	display: flex;
	align-items: center;
	font-feature-settings: "pnum" on, "lnum" on;
	font-size: 14px;
	font-weight: 500;
	line-height: 21px;
	& input[type="radio"] {
		margin-right: 15px;
	}
}
