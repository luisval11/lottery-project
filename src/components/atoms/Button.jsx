import { h } from "preact";

export const Button = ({ text, ...rest }) => {
	return <button {...rest}>{text}</button>;
};
