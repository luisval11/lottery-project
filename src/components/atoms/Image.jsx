import { h } from "preact";

export const Image = ({src, ...rest}) => {
	return <img src={src} {...rest} />;
};
