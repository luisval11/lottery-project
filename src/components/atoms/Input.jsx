export const Input = ({type, ...rest}) => {
	return (
		<input type={type} {...rest}/>
	);
};
