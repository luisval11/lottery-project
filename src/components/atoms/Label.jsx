import { h } from "preact";
export const Label = ({ children, text, ...rest }) => {
	return (
		<label {...rest}>
			{children}
			{text}
		</label>
	);
};
