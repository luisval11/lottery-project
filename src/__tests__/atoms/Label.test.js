import { h } from "preact";
import { mount } from "enzyme";
import { Label } from "../../components/atoms/Label";

describe("Test on Label", () => {
	test("should match initial snapshot", () => {
		let wrapper = mount(
			<Label text={"Label"}/>
		);
		expect(wrapper).toMatchSnapshot();
	});

	test("should have a value 'label' for text", () => {
		const textParam = "label";
		let wrapper = mount(<Label text={textParam} />);
		expect(wrapper.find('label').text().trim()).toBe(textParam);
	});
});
