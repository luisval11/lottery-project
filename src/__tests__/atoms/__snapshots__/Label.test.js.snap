// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on Label should match initial snapshot 1`] = `ReactWrapper {}`;
